ds3-autogen-java/src/main/resources/tmpls/physical_placement_request_template.tmpl

```diff
@@ -3,19 +3,45 @@
 package ${packageName};
 
 import com.spectralogic.ds3client.HttpVerb;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.models.bulk.Ds3ObjectList;
+import com.spectralogic.ds3client.serializer.XmlOutput;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.List;
 <#include "imports.tmpl"/>
 
 public class ${name} extends AbstractRequest {
 
     // Variables
+    private final List<Ds3Object> objects;
     <#include "variables.tmpl"/>
 
     // Constructor
-    <#include "constructor.tmpl"/>
+    public ${name}(${javaHelper.constructorArgs(javaHelper.addArgument(requiredArguments, "Objects", "List<Ds3Object>"))}) {
+        this.objects = objects;
+        <#list requiredArguments as arg>
+        this.${arg.getName()?uncap_first} = ${arg.getName()?uncap_first};
+        </#list>
+        <#if operation??>
+        this.getQueryParams().put("operation", "${operation.toString()?lower_case}");
+        </#if>
+    }
 
     <#include "with_constructors.tmpl"/>
 
+    public Stream getContentStream() {
+        final Ds3ObjectList objects = new Ds3ObjectList();
+        objects.setObjects(this.objects);
+        final String xmlOutput = XmlOutput.toXml(objects);
```

Comment:
This will print the size with the payload, which we do not want.  You need to use one of the other toXml methods that checks to see if the request is a bulk put request.